Dockerfile

```diff
@@ -0,0 +1,49 @@
+############################
+# STEP 1 build executable binary
+############################
+FROM golang:1.13-alpine AS builder
+
+RUN apk update && apk add --no-cache git
+
+ARG COMMIT_SHA=<not-specified>
+
+# Create appuser.
+RUN adduser -D -g '' appuser
+
+WORKDIR /app
+COPY . .
+
+RUN echo "$COMMIT_SHA" >> ./commit.sha
+
+RUN go mod download
+RUN go mod verify
+
+RUN GOOS=linux CGO_ENABLED=0 GOARCH=amd64 go build -ldflags="-w -s" -o main .
+
+############################
+# STEP 2 build service image
+############################
+
+FROM scratch
+
+LABEL maintainer="%CUSTOM_PLUGIN_CREATOR_USERNAME%" \
+  name="%CUSTOM_PLUGIN_SERVICE_NAME%" \
+  description="%CUSTOM_PLUGIN_SERVICE_DESCRIPTION%" \
+  eu.mia-platform.url="https://www.mia-platform.eu" \
+  eu.mia-platform.version="1.1.1-rc.0"
```

Comment:
Why version 1.1.1-rc.0? How can we update this?

Reply:
I forgot to change it, I'll put version 0.1.0 as the other templates